### DISCUSSION TITLE
regulon score seen in the .p fildes

### DISCUSSION THREAD

**Discussion by GeneVector5** [aertslab/scenicplus#205] [Q&A]
For pySCENIC and SCENIC+, what is the meaning of the regulon score and why is not strictly between 0 and 1? I have seen cases where the value is about 1

**Comment by SeppeDeWinter** [comment]
Hi @GeneVector5

I transfered your question to the discussion section.

Can you specify which scores you are talking about? We have the AUC values for regulons but also the regulon specificity scores (RSS)

In case you are talking about AUC values: These are area under the curve values. In the case of the target genes, these values are generated by ranking each gene based on their expression value in each cells (from highest expression to lowest). Then a recovery curve is generated (similar to GSEA) representing how many of the target genes of each regulon are found at each position of the ranking. Finally the area under the curve is calculated up until the top 5 % (by default) of the ranking. The units of this value are arbitrary, and the values are not necessarily between 0 and 1.

In case you are talking about the RSS values. These values should be between 0 and 1. The value is the jensen-shannon distance between the actual distribution of AUC values across cells and the fictional distribution if the regulon would only be activate in a single cell type. This metric came from this publication from Suo et al.: https://www.sciencedirect.com/science/article/pii/S2211124718316346?via%3Dihub#sec4 (see subheading **Quantifying cell-type specificity score**). 

Best,

Seppe
